src/stitches.config.ts

```diff
@@ -0,0 +1,301 @@
+import { createStitches } from '@stitches/react';
+import type * as Stitches from '@stitches/react';
+export type { VariantProps } from '@stitches/react';
+
+export const MediaQueryKeys = {
+  xs: '(max-width: 520px)',
+  sm: '(max-width: 900px)',
+  md: '(max-width: 1200px)',
+  lg: '(max-width: 1800px)',
+} as const;
+
+export const {
+  styled,
+  css,
+  globalCss,
+  keyframes,
+  getCssText,
+  theme,
+  createTheme,
+  config,
+} = createStitches({
+  theme: {
+    colors: {
+      white: '#fff',
+      black: '#000',
+      // red
+      red200: '#F5E4E4',
+      red300: '#E6BCBC',
+      red400: '#EF7376',
+      red500: '#6f3638',
+      // gray
+      gray400: '#B5BBBD',
+
+      lightBackground: '#E1E1E1',
+      subtleGray: '#F5F5F5',
+      almostWhite: '#F6F7F8',
+      third: '#EAEAEB',
+      gray: '#EFF3F4',
+
+      lightBorder: '#DEDEDE',
+      lightGray: '#C3CDCF',
+      mediumGray: '#A7B0B4',
+      lightText: '#859296',
+      border: '#939EA1',
+      placeholder: '#99A2A5',
+      placeholderDark: '#7C898D',
```

Comment:
This is the sort of thing that we can simplify, maybe it is something like grey400@40%

However we do this, I think it would be best if we didn't stoop to the level of the poor way it's been done in the past.

Ideally there is something like the design tokens so that we can easily do themes in the future.

Reply:
I found a screenshot, I shared for this point below

Reply:
Sure, I guess first we need to get the rights colors system, for now just leave as we have defined on material-colors